corehq/util/celery_utils.py

```diff
@@ -186,3 +190,45 @@ def get_running_workers(timeout=10):
         worker_names.extend(worker_info.keys())
 
     return worker_names
+
+
+class LoadBasedAutoscaler(Autoscaler):
+    def _maybe_scale(self, req=None):
+        procs = self.processes
+        cur = min(self.qty, self.max_concurrency)
+
+        available_cpus = multiprocessing.cpu_count()
+        load_avgs = os.getloadavg()
+        max_avg = max(load_avgs[0], load_avgs[1])
+        normalized_load = max_avg / available_cpus
+
+        if cur > procs and normalized_load < 0.90:
+            self.scale_up(cur - procs)
+            return True
+        elif cur < procs:
+            self.scale_down((procs - cur) - self.min_concurrency)
```

Comment:
Thanks. took me a minute to get my head around how that could happen. https://github.com/dimagi/commcare-hq/pull/16967/commits/413c07f378903454da212d46c398a914d925cb42

FWIW this bug is also in the default autoscaler (i also checked the latest version of celery 3.1.25 and it wasn't fixed). It's possible we've never run into it because all of our mins are 0